### ISSUE TITLE
Voiceover doesn't read tabs

### ISSUE THREAD

**Issue by sana-ajani** [microsoft/vscode#88729]
Steps to Reproduce:

1. Open new file, write some lines of text
2. On new line, press tab a few times. Nothing is read by Voiceover, expected behavior is to read "tab". 

![tab_voiceover](https://user-images.githubusercontent.com/9721254/72483054-92843400-37b4-11ea-8b13-b99e95aad505.gif)

Version: 1.42.0-insider
Commit: 5e758c10c969d668730f036abfc557d9cfb62b87
Electron: 7.1.7
OS: MacOS Mojave

@ivanfetch

**Comment by isidorn** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/70033
Since that line is empty nothing is read out.
On a line with content once I press Tab voiceover reads Tab nicely for me

**Comment by ivanfetch** [comment]
@sana-ajani Is this issue intended to capture that Visual Studio Code does not reflect the indent level using VoiceOver? I demonstrated on our call using Vim as an example showing the indent level as a number in the left margin (which VOiceOver speaks)

A feature that verbally reflects the indent level while using the arrow-keys to navigate code is very valuable, especially when working with YAML or Python which requires indenting as part of the syntax.

**Comment by isidorn** [comment]
@ivanfetch thanks for coming back. Yeah the issue did not have any details about the indent reading. can you please create a new issue that captures that and ping me @isidorn on it? Thanks a lot!

**Comment by ivanfetch** [comment]
For reference, I've created issue #88877

**Comment by sana-ajani** [comment]
Sorry @ivanfetch, the indentation demo you showed is what I was trying to capture. Thanks for creating the other issue!
